src/manager.js

```diff
@@ -36,42 +39,44 @@ Promise.all([bookingData, roomData, userData]).then((requiredData) => {
   $('.date').html(manager.todaysDate());
   $('.revenue').html(manager.getTotalSpend(bookedTodayData));
   $('.percentage').html((bookedTodayData.length/manager.rooms.length) * 100)
-  manager.availableToday(bookedTodayData).forEach(room => {$('.available').append(`<span class="rooms">Room:${room.number} Beds: ${room.numBeds} ${room.bedSize.toUpperCase()} Price: $${room.costPerNight}</span>`)})
+  manager.availableToday(bookedTodayData).forEach(room => {$('.available').append(`<button class="rooms">Room:${room.number} Beds: ${room.numBeds} ${room.bedSize.toUpperCase()} Price: $${room.costPerNight}</button>`)})
 }).catch(data => console.log('Fetch error', data))
 
 $('#search').keyup(() => {
+  let search = $('#search').val()
   $('#search-results').html('')
-  manager.customerSearch($('#search').val()).forEach(customer => {
-    $('#search-results').append(`<span id="${customer.id}" class="upcoming-rooms">Name: ${customer.name}</span>`)
+  manager.customerSearch(search.toLowerCase()).forEach(customer => {
+    $('#search-results').append(`<button id="${customer.id}" class="upcoming-rooms">Name: ${customer.name}</button>`)
   })
 })
 
 $('#search-results').click(event => {
   $('.bookings').html(`<span data-id="${event.target.id}" class="spending"></span>`)
   if (event.target.id > 0 && event.target.id <= 50) {
-    manager.getMyBookings(parseInt(event.target.id)).forEach(room => {$('.bookings').append(`<p class="upcoming-rooms" data-date="${room.date}" data-bookingid="${room.id}">Room: ${room.roomNumber} ${room.date}</p>`)})
+    manager.getMyBookings(parseInt(event.target.id)).forEach(room => {$('.bookings').append(`<button class="upcoming-rooms" data-date="${room.date}" data-bookingid="${room.id}">Room: ${room.roomNumber} ${room.date}</button>`)})
```

Comment:
This was really smart to change these to buttons.  I have mine wrapped in a div but a button is semantically appropriate and easy to style.  I may implement this as well.